tests/PerformanceTests/Microsoft.Identity.Web.Perf.Client/TokenCacheHelper.cs

```diff
@@ -14,7 +14,7 @@ namespace Microsoft.Identity.Web.Perf.Client
     /// Token cache writing on disk one cache per account
     /// WARNING: this version is not encrypted
     /// </summary>
-    static class ScalableTokenCacheHelper
+    static class TokenCacheHelper
```

Comment:
why change its name?

Reply:
That was feedback from one of the previous PRs.